packages/data-explorer-ui/.storybook/main.ts

```diff
@@ -1,19 +1,30 @@
-const path = require("path");
-const toPath = (filePath) => path.join(process.cwd(), filePath);
+import path from "path";
 
-module.exports = {
+const toPath = (filePath: string) => path.join(process.cwd(), filePath);
+
+export default {
   stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
   addons: [
     "@storybook/addon-links",
     "@storybook/addon-essentials",
     "@storybook/addon-interactions",
     "@storybook/addon-mdx-gfm",
   ],
+  typescript: {
+    reactDocgen: "react-docgen-typescript",
+    reactDocgenTypescriptOptions: {
+      // Filter out third-party props from node_modules except @mui packages
+      propFilter: (prop: any) =>
```

Comment:
This change allows us to automatically inject MUI fields to story if a component inherits them (but sadly only default versions, ignoring overrides or custom colors)